### PR TITLE
Fix the unit-test due to changes in the TimestampClosable contract.

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

**Comment by jwang47** [comment]
LGTM

File: cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/StreamDataFileTestBase.java [hunk 1]
```diff
@@ -875,8 +875,13 @@ public void testEventTemplate() throws IOException, InterruptedException {
...
-    // The close timestamp should be -1 before the writer is closed.
-    Assert.assertEquals(-1L, writer.getCloseTimestamp());
+    // Trying to get close timestamp should throw exception before the file get closed
+    try {
+      writer.getCloseTimestamp();
+      Assert.fail();
+    } catch (IllegalStateException e) {
+      // Expected
+    }
     writer.close();
```
Comment (bdmogal) [inline cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/StreamDataFileTestBase.java]: Should you assert the close timestamp after closing?

**Comment by abaranau** [comment]
LGTM

**Comment by bdmogal** [comment]
Minor comment, other than that :+1: Tested locally.

File: cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/StreamDataFileTestBase.java (hunk 1 above)
Comment (chtyim) [inline cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/StreamDataFileTestBase.java]: the `getCloseTimestamp` is used below.

Comment (bdmogal) [inline cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/StreamDataFileTestBase.java]: Yup, missed it. ignore my comment.

**Pull request merged**